apps/condo/domains/user/utils/directAccess/config.js

```diff
@@ -23,6 +23,7 @@ const DIRECT_ACCESS_AVAILABLE_SCHEMAS = {
         'B2CAppAccessRight',
         'B2CAppBuild',
         'B2CAppProperty',
+        'Ticket',
```

Comment:
Please add Domain-comment-separator and keep things alphabetically order.

P.S.: Right now it's looking clean, but once most of domains will be migrated it'll became hard to read (remember translations mess), so alphabetic domains + alphabetic inside 🙏

Reply:
done